v4/export/sql.go

```diff
@@ -209,6 +234,17 @@ func GetPrimaryKeyName(db *sql.DB, database, table string) (string, error) {
 	return colName, nil
 }
 
+func GetUniqueIndexName(db *sql.DB, database, table string) (string, error) {
+	uniKeyQuery := fmt.Sprintf("SELECT column_name FROM information_schema.columns " +
+		"WHERE table_schema = `%s` AND table_name = `%s` AND column_key = 'UNI';", database, table)
+	var colName string
+	row := db.QueryRow(uniKeyQuery)
```

Comment:
Take care. When `QueryRow` doesn't get a result it will return a `sql.ErrNoRows` error.

Reply:
```
// QueryRow executes a query that is expected to return at most one row.
// QueryRow always returns a non-nil value. Errors are deferred until
// Row's Scan method is called.
// If the query selects no rows, the *Row's Scan will return ErrNoRows.
// Otherwise, the *Row's Scan scans the first selected row and discards
// the rest.
```
seems it will return error in `row.Scan`

Reply:
Yes

Reply:
do you check it?